app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,10 @@
 class RegistrationsController < Devise::RegistrationsController
   respond_to :json
 
-  # before_action :configure_sign_up_params, only: [:create]
+  # rubocop:disable Rails/LexicallyScopedActionFilter
+  before_action :configure_sign_up_params, only: [:create]
+  # rubocop:enable Rails/LexicallyScopedActionFilter
```

Comment:
I'm curious why we need to disable rubocop and then enable it again here.

Reply:
This is because the create action is not defined in this file. It is inherited. And rubocop complains about that because it thinks its not within scope, but it is.

Reply:
https://www.rubydoc.info/gems/rubocop/0.58.0/RuboCop/Cop/Rails/LexicallyScopedActionFilter

Reply:
I guess rubocop wants this before_action defined on the superclass where the create method exists, but that class is not in our application so we can't do that. So I ignore it.